vividus-extension-selenium/src/test/java/org/vividus/steps/ui/GenericElementStepsTests.java

```diff
@@ -107,6 +109,29 @@ void shouldThrowAnExceptionIfLocatorVisibilityAndStateToCheckAreTheSame(Visibili
                 visibility, state), iae.getMessage());
     }
 
+    @ParameterizedTest
+    @CsvSource({ "VISIBLE, NOT_VISIBLE",
+                 "INVISIBLE, VISIBLE" })
+    void shouldThrowAnExceptionIfLocatorVisibilityAndStateToCheckAreDifferent(Visibility visibility, State state)
+    {
+        Locator locator = getLocatorWithVisibility(visibility);
+        IllegalArgumentException iae = assertThrows(IllegalArgumentException.class,
```

Comment:
not critical, just a hint: you can use `var` keyword in unit tests:
```suggestion
        var illegalArgumentException = assertThrows(IllegalArgumentException.class,
```